sql/mysql_db/mysql_db.go

```diff
@@ -633,16 +671,13 @@ func (db *MySQLDb) UserHasPrivileges(ctx *sql.Context, operations ...sql.Privile
 			if tblSet.Has(operationPriv) {
 				continue
 			}
-			colSet := tblSet.Column(operation.Column)
```

Comment:
We're removing column checks? I know they're not fully supported, but why wouldn't we leave this in place?